project/{{ cookiecutter.__folder_name }}/devops/inventory/group_vars/all/projects.yml

```diff
@@ -1,7 +1,7 @@
 ---
 project_folders:
   # Path to store {{ cookiecutter.hostname }} postgres data
-    path: "{{ lookup('ansible.builtin.env', 'DEPLOY_FOLDER', default='/srv/plone/data') }}"
+  - path: "{{ lookup('env', 'DEPLOY_FOLDER', default='/srv/plone/data') }}"
```

Comment:
Removing the dash doesn't look right to me. `project_folders` is used as a list here: https://github.com/plone/cookieplone-templates/blob/185dfb4f6589c60b522cb6cd7c06397be0d64710/project/%7B%7B%20cookiecutter.__folder_name%20%7D%7D/devops/playbooks/setup.yml#L134

Reply:
@erral Okay, thanks for testing it!